### PR TITLE
Cleanup unused colors reported on css-stats

### PR DESCRIPTION
**Pull request by wfortin**
AW YIS!

According to http://cssstats.com/stats?link=http%3A%2F%2Fcoveo.github.io%2Fvapor%2Fdist%2Fcss%2FCoveoStyleGuide.css

I reviewed and removed/updated the colors.

Ditched commons/colors, we'll now only use palette.scss

### CODE REVIEW

**Comment by JPLachance** [comment]
Plz bring back the stratos color :P

File: scss/tables/table.scss [hunk 1]
```diff
@@ -306,8 +306,8 @@ table {
 
   &.confirm-delete {
     .row-selected td, &.row-selected:hover td {
-      background-color: $invalid-background-color;
-      border-color: $invalid-message-background-color;
+      background-color: $white;
```
Comment (JPLachance) [inline scss/tables/table.scss]: Hum, nope. $invalid-background-color was a light red, and you changed it for $white (a light gray). Not sure we want to change that.

Comment (JPLachance) [inline scss/tables/table.scss]: You may set $red with an opacity of like 20%.

Comment (wfortin) [inline scss/tables/table.scss]: I'll go with that.

**Comment by GermainBergeron** [comment]
:shipit:

**Pull request merged**